src/Xamarin.Android.Build.Tasks/Tasks/MakeBundleNativeCodeExternal.cs

```diff
@@ -23,7 +23,6 @@ public class MakeBundleNativeCodeExternal : AndroidTask
 
 		const string BundleSharedLibraryName = "libmonodroid_bundle_app.so";
 
-		[Required]
 		public string AndroidNdkDirectory { get; set; }
```

Comment:
I removed `[Required]` because the `NdkTools.Create()` call below gives a much better error message:

    error XA5104: Could not locate the Android NDK. Please make sure the Android NDK is installed in the Android SDK Manager, or if using a custom NDK path, please ensure the $(AndroidNdkDirectory) MSBuild property is set to the custom path.